setup_configure.py

```diff
@@ -86,12 +89,18 @@ class configure(Command):
 
     user_options = [('hdf5=', 'h', 'Custom path to HDF5'),
                     ('hdf5-version=', '5', 'HDF5 version "X.Y.Z"'),
+                    ('hdf5-includedir=', 'i', 'path to HDF5 headers'),
+                    ('hdf5-libdir=', 'l', 'path to HDF5 library'),
+                    ('hdf5-pkgconfig-name=', 'p', 'name of HDF5 pkgconfig file'),
```

Comment:
It looks like the added config options are no longer required for the MPI testing. Can I suggest that we pull these additions out as a separate PR, and do the minimum necessary in this one to effectively test with MPI?

Reply:
Done, now in #1414.